packages/datadog-plugin-mysql/src/index.js

```diff
@@ -55,4 +56,14 @@ class MySQLPlugin extends Plugin {
   }
 }
 
+function getServiceName (tracer, config, dbConfig) {
+  if (typeof config.service === 'function') {
+    return config.service(dbConfig)
```

Comment:
There should be a test for this.

Reply:
Added tests to confirm this is called with correct parameters for both `mysql` and `mysql2`